sdk/appcenter-analytics/src/main/java/com/microsoft/appcenter/analytics/Analytics.java

```diff
@@ -41,7 +41,7 @@
      * Max length of properties.
      */
     @VisibleForTesting
-    static final int MAX_PROPERTY_ITEM_LENGTH = 64;
```

Comment:
The limits are also in the doc of the trackEvent/Page  methods see `public static void trackEvent(String name)`